osbuild/buildroot.py

```diff
@@ -275,6 +275,7 @@ def run(self, argv, monitor, timeout=None, binds=None, readonly_binds=None):
 
         # Setup a new environment for the container.
         env = {
+            "CONTAINER": "bwrap-osbuild",
```

Comment:
Should it be `container` (lower case)? Or am I reading the `CONTAINER_INTERFACE` doc wrong?